### DISCUSSION TITLE
Motor Problem

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#819] [Q&A]
Hi, when I use commander to take off. I saw my crazyflie happen some error issue. In the beginning, the motro is normal but few second later the motor start drop the RPM and when I modify that attitude the motor will increase but another decrease RPM. I know the quadcopter basic knowledge is modify motor RPM to create attitude. but in the case, the RPM is drop to 0. I think it not normal.  I have the test video, but seems like github could't to upload.

**Comment by ataffanel** [comment]
Could you describe a bit more your setup?
 - Are you using a positioning system, is it working properly (can you verify that the position is well estimated)
 -  Is the Crazyflie free to move or held somehow (what you describe is a behavior that happens when the Crazyflie is held while tying to control its attitude)
 - Is the Crazyflie flying well when flown manually with a gamepad?

**Reply by CalvinKoung** [reply]
I hold my hand and use client command button to control crazyflie to takeoff and land. Can I sent the video via the E-mail to you ? I think this can be specific. thanks

**Reply by CalvinKoung** [reply]
Hi, I screen shot the motor of plotter. I hold the crazyflie on the table but few second ago can saw the motor1 drop to 0 (I tried few times sometime will happened in motor3). 
![90D4CDB0-060C-403C-BFDA-062A98E0D9DF](https://github.com/bitcraze/discussions/assets/118436188/3fe04b09-53d2-4fb5-a2fa-a4cf3d36298f)

**Reply by ataffanel** [reply]
If you are holding the Crazyflie, this is a normal behavior: the Crazyflie will try to control its attitude. If you are blocking it the Integral part of the controller will grow enough to saturate the motors. For one side it means full thrust, for another side it means the motor will stop.

So this is not necessary a problem. I suggest you fly the Crazyflie manually to see if it is able to take-off by itself.

**Reply by CalvinKoung** [reply]
Ok, I got it. I will tried it. Thanks your help.
